src/as3hx/Writer.hx

```diff
@@ -1306,6 +1306,14 @@ class Writer
                 
                 lvl++;
                 for(c in newCases) {
+
+                    if(def != null &&
+                        def.before != null &&
+                        def.before.el.toString() == c.el.toString()) {
```

Comment:
I don't really like this `el.toString()` comparison, but it works. Any better ideas to compare `el`?